src/Uno.UI.Wasm/ts/Interop/AsyncInteropHelper.ts

```diff
@@ -0,0 +1,38 @@
+﻿namespace Uno.UI.Interop {
+	export class AsyncInteropHelper {
+
+		private static dispatchResultMethod: (handle: number, result: string) => string;
+		private static dispatchErrorMethod: (handle: number, error: string) => string;
+
+		private static init() {
+			if (AsyncInteropHelper.dispatchErrorMethod) {
+				return; // already initialized
+			}
+			const w = window as any;
+			AsyncInteropHelper.dispatchResultMethod =
+				w.Module.mono_bind_static_method("[Uno.Foundation] Uno.Foundation.WebAssemblyRuntime:DispatchAsyncResult");
+			AsyncInteropHelper.dispatchErrorMethod =
+				w.Module.mono_bind_static_method("[Uno.Foundation] Uno.Foundation.WebAssemblyRuntime:DispatchAsyncError");
+		}
+
+		public static Invoke(h: number, m: () => Promise<string>): void {
```

Comment:
Is there a perf consideration with short names?

Reply:
Nope.  Why?